tslint.json

```diff
@@ -8,7 +8,12 @@
     "interface-name": [true, "never-prefix"],
     "no-shadowed-variable": false,
     "no-string-literal": false,
-    "trailing-comma": false,
-    "member-ordering": ["fields-first"]
+    "trailing-comma": [false],
+    "member-ordering": ["fields-first"],
+    "array-type": ["array-simple"],
+    "arrow-parens": false,
+    "max-classes-per-file": [false],
+    "prefer-for-of": false,
+    "ban-types": false

```

Comment:
I am not sure we needed this rule to be enabled, and it was turned on after the tslint update. I turned it off for now as I did not see a good way to refactor many of the constructors that were using `Function` as a type.